packages/@sanity/block-tools/README.md

```diff
@@ -159,11 +159,12 @@ const partialBlock = {
   children: [
     {
       _type: 'span',
-      text: 'Foobar'
+      text: 'Foobar',
+      marks: ['strong', 'df324e2qwe']
     }
   ]
 }
-normalizeBlock(partialBlock)
+normalizeBlock(partialBlock, {decorators: ['strong']})
```

Comment:
If I understand this correctly, this option should actually be named `allowedDecorators` - right now it's a bit hard to know what this option does.

Reply:
Yeah, I agree, that's a more sound name.